models/mapper_test.go

```diff
@@ -170,6 +179,17 @@ func TestRectifyKeywords_EightKeywordsAndTenAsLimit(t *testing.T) {
 	})
 }
 
+func TestValidate_WithNiltopics(t *testing.T) {
```

Comment:
You should also do another test case for when no topics are given (i.e when the array is empty - `len = 0`)